### PR TITLE
Fix typos in APIReviewProcess.md

### PR DESCRIPTION
**Pull request by GiovanniBraconi**
- [x] You've read the [Contributor Guide](https://github.com/dotnet/aspnetcore/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/dotnet/aspnetcore/blob/main/CODE-OF-CONDUCT.md).
- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

## Description

Fixes typos in APIReviewProcess.md

### CODE REVIEW

File: docs/APIReviewProcess.md [hunk 1]
```diff
@@ -40,13 +40,13 @@ In general, larger changes should have more explanation and context provided, an
...
 
 We use the ref-assembly format because it's more readable and useful for the kinds of things that come up in api-review discussions. Using a more compact format (without docs and implementations) makes it easier to notice patterns. In the rare case that you have to manually transcribe this format, think of this as you spending a little time to save a lot of others time in the meeting.
 
 ## If you are the "champion" for a community-submitted change
 
-If you are assigned a community-submitted change to *champion* in our API-review then just put on your pretend pajamas and pretend that it was your change to begin with. Come to the meeting ready to explain why this addition is needed, and why it's the best approach.
+If you are assigned a community-submitted change to *champion* in our api-review then just put on your pretend pajamas and pretend that it was your change to begin with. Come to the meeting ready to explain why this addition is needed, and why it's the best approach.
```
Comment (mkArtakMSFT) [inline docs/APIReviewProcess.md:49]:
```suggestion
If you are assigned a community-submitted change to *champion* in our API-review, then just put on your pretend pajamas and pretend that it was your change to begin with. Come to the meeting ready to explain why this addition is needed, and why it's the best approach.
```

Comment (mkArtakMSFT) [reply docs/APIReviewProcess.md]: This one should have been left as is.

**Review by mkArtakMSFT — APPROVED** [review]
Thanks @GiovanniBraconi !

**Comment by danmoseley** [comment]
Any further typos could you if possible use a single Pr?

**Comment by danmoseley** [comment]
(Saving validation machine time)

**Pull request merged**